### DISCUSSION TITLE
How to enable kind: ScrapeConfigs?

### DISCUSSION THREAD

**Discussion by abalage** [prometheus-operator/kube-prometheus#2105] [General] [answered]
Hello,

ScrapeConfigs CRD has been added recently to https://github.com/prometheus-operator/prometheus-operator/pull/5335
Probably a next release would contain it but I'd like to test it in advance.
I have built the manifests from main branch with my other customization by using jsonnet.
However the resulted manifests do not contain the aforementioned CRD.
Is there anything special which needs to be added?

ps: Do you know any eta when the next release will be cut?
ps2: Sorry if this is not the right place to ask for this.

Kind regards,

**Comment by simonpasquier** [comment, marked as answer]
this should be fixed by https://github.com/prometheus-operator/prometheus-operator/pull/5574
this will be included in prometheus-operator v0.66.0 but you can pull it from the github.com/prometheus-operator/prometheus-operator's main branch already.

**Reply by abalage** [reply]
Thanks for the reply.
